node_modules/oae-core/documentpreview/js/documentpreview.js

```diff
@@ -108,14 +108,14 @@ define(['jquery', 'underscore', 'oae.core', 'lazyload'], function($, _, oae) {
             page.height = page.$el.height();
             page.marginTop = CONTENT_PAGE_SPACING;
 
-            // Request the page content
+            // Request the page content HTML version
             $.ajax({
```

Comment:
Thinking about this, my preference would be to create a new endpoint that returns both the html and txt content as properties

```
html: "...",
text: "..."
```

If this is controlled by the signatures as well, we wouldn't have to worry about one of them expiring.

Would you be able to create a ticket for this and assign to @simong ?